lib-cloudsdk/src/main/java/com/facebook/openwifi/cloudsdk/models/ap/State.java

```diff
@@ -132,4 +132,5 @@ public static class Radio {
 	@SerializedName("link-state") public JsonObject linkState;
 	public JsonObject gps;
 	public JsonObject poe;
+	public long timeStamp;
```

Comment:
this should NOT be added here if it isn't present in the model coming from OWF - define a subclass like we did for wifiscans (`WifiScanEntry` vs. `WifiScanEntryResult`)